pkg/cmd/secvulnMaven.go

```diff
@@ -5,33 +5,287 @@
 package cmd
```

Comment:
copy right for go is:-

/*
 * Copyright contributors to the Galasa project
 */